pyfakefs/fake_file.py

```diff
@@ -352,7 +352,7 @@ def size(self, st_size: int) -> None:
     @property
     def path(self) -> AnyStr:
         """Return the full path of the current object."""
-        names: List[AnyStr] = []
+        names: List[AnyStr] = []  # pytype: disable=invalid-annotation
```

Comment:
Most of the problems are related to `AnyStr`, which seems to behave differently with `pytype` compared to `mypy` - still have to figure that out.

Reply:
Oh, egg on my face... it does not include None. I'm still not sure why path would return bytes. I see where AnyStr is useful for setting contents on a new file, but not why it is supported for anything else. Is it possible to implement the same annotations as implemented in the standard library?

Reply:
Again, answering my own question, I should really do some research before asking them, this annotation is close to the typeshed annotations. https://github.com/python/typeshed... but it might be a good idea to import them directly, if possible... for future investigation.

Reply:
(this was in response to your previous comment, I hadn't seen your own response yet)

Reply:
I'm going to merge this as is now with the hope that the pytpe support can be improved with time.